doc/nrf/ug_gzll.rst

```diff
@@ -70,19 +65,39 @@ The radio and timer interrupt handlers should run at priority level 0 (highest p
 To avoid blocking Gazell operations, applications can run at priority level 2 or higher.
 
 You can customize Gazell at runtime for a range of different applications.
-See the :ref:`nrfxlib:gzll_api` for a list of configuration functions as well as the default and constant parameters.
+See the :ref:`nrfxlib:gzll` and :ref:`nrfxlib:gzll_api` for a list of configuration functions as well as the default and constant parameters.
+
+.. note::
 
-.. note:
    Editing the header file containing the default and constant parameters does not change their value when compiling a new project.
    These values are provided as a useful reference when making an application with the precompiled library.
 
 Gazell modes
 ************
 
-A member of a Gazell star network is either a Host or Device.
-Up to eight Devices can communicate with a single Host.
-Each Host can communicate with up to eight Devices.
-Each Device communicates to a single Host.
+Gazell has the following two modes available:
+
+* Device: an initiator mode, where the Device transmits packets periodically.
+* Host: a listening mode, where the Host works mainly as a receiver and only transmits ACK packets back to the Device.
+  The ACK packets can optionally carry payload (enabling duplex communication).
+
+One application can only work in a single Gazelle mode at a time.
+However, the mode can be switched in runtime.
+
+
+A member of a Gazell star network is either a Host or Device, and up to eight Devices can communicate with a single Host.
+Each Host can communicate with up to eight Devices, and each Device communicates to a single Host.
+
+
+
+
+.. DRAFT!! Version 2:
+
+>>>>>>> 649cb123d (doc improvement: Gazell)
```

Comment:
Something must have gone wrong with the rebase (fixup) as you still have these >>> chars. And the DRAFT thing.